lis/dataassim/obs/S1_sigmaVVSM/S1_sigmaVVSM_Mod.F90

```diff
@@ -268,26 +259,16 @@ subroutine S1_sigmaVVSM_setup(k, OBS_State, OBS_Pert_State)
 ! set up the S1 domain %and interpolation weights. 
 !-------------------------------------------------------------
 
-    ! Open first file to get nx ny dimensions
-    call system('ls ./' // trim(S1sigmaobsdir) // ' > ./S1_listfiles.txt')
-
-    open(flist, file=trim('./S1_listfiles.txt'), &
-         status='old', iostat=status)
-
-    read(flist, '(a)', iostat=status) S1_firstfile
-    S1_filename = trim(S1sigmaobsdir) // '/' // trim(S1_firstfile)
-
-    ios = nf90_open(path=S1_filename,&
-                    mode=NF90_NOWRITE,ncid=ncid)
-    call LIS_verify(ios,&
-        'Error reading in S1 data dimensions: Error opening file '// S1_filename)
-    ios = nf90_inquire_dimension(ncid,1,yname,NY)
-    ios = nf90_inquire_dimension(ncid,2,xname,NX)
-    ios = nf90_close(ncid)
-
     do n=1,LIS_rc%nnest
-       S1_sigma_struc(n)%nc = NX
-       S1_sigma_struc(n)%nr = NY
+       call ESMF_ConfigFindLabel(LIS_config,&
+                    "S1 backscatter domain x-dimension size:",rc=status)
+       call ESMF_ConfigGetAttribute(LIS_config,S1_sigma_struc(n)%nc,rc=status)
+       call LIS_verify(status,'[ERR] S1 backscatter domain x-dimension size: not defined')
```

Comment:
Perhaps specify S1 *VV* backscatter, if we really use this entire module only for VV? (to be updated on all occasions then)